libs/api-types/src/user.ts

```diff
@@ -62,3 +62,15 @@ export interface UserData {
   pullingDataOtherAPIS?: string
   apiTerms?: string
 }
+
+export interface UserFetchResponse {
```

Comment:
We already have a [paginated type with a Generic](https://github.com/GlobalFishingWatch/frontend/blob/develop/libs/api-types/src/pagination.ts#L1) to define the entries types, I think I'd remove this to avoid duplications